src/generic/grideditors.cpp

```diff
@@ -703,6 +707,40 @@ void wxGridCellNumberEditor::Create(wxWindow* parent,
     }
 }
 
+void wxGridCellNumberEditor::SetSize(const wxRect& r)
+{
+#if wxUSE_SPINCTRL
+    if ( HasRange() )
+    {
+        wxASSERT_MSG(m_control, "The wxSpinCtrl must be created first!");
+
+        const wxSize bestSize = Spin()->GetBestSize();
+        const int width = wxMax(r.GetWidth(), bestSize.GetWidth());
+        const int height = bestSize.GetHeight() > 0 ?
+                           bestSize.GetHeight() :
+                           r.GetHeight();
```

Comment:
This seems to be unused. It was probably meant to be used in the line just below?